WordPress/src/main/java/org/wordpress/android/ui/pages/PagesFragment.kt

```diff
@@ -246,13 +246,17 @@ class PagesFragment : Fragment(), GutenbergWarningDialogClickInterface {
         viewModel.editPage.observe(this, Observer { page ->
             page?.let {
                 val post = postStore.getPostByLocalPostId(page.pageId)
-                val isGutenbergContent = PostUtils.contentContainsGutenbergBlocks(post?.content)
-                if (isGutenbergContent && !AppPrefs.isGutenbergWarningDialogDisabled()) {
-                    PostUtils.showGutenbergCompatibilityWarningDialog(
-                            getActivity(), fragmentManager, post, viewModel.site
-                    )
+                if (PostUtils.shouldShowGutenbergEditor(false, post)) {
```

Comment:
Feels to me that the check for Gutenberg editor is bleeding into this class (`PagesFragment`) but it'd better not. Can we leave it out, perhaps by having it inside `ActivityLauncher.editPageForResult()`?

Reply:
see comment above